src/libraries/Microsoft.PowerFx.Interpreter/Functions/Mutation/PatchFunction.cs

```diff
@@ -218,7 +218,7 @@ public override bool CheckInvocation(TexlBinding binding, TexlNode[] args, DType
 
                 foreach (var typedName in curType.GetNames(DPath.Root))
                 {
-                    if (!tableType.HasField(typedName.Name))
+                    if (!tableType.TryGetFieldType(typedName.Name, out var displayName, out _))
```

Comment:
>TryGetFieldType [](http://example.com/codeflow?start=35&length=15)

was this intentional? 
should keep HasField for perf reasons, or check with Luc.

Reply:
It wasn't. I had to deal with some git issues and this ended up being pushed. Thanks for catching this